IntegrationTests/UI.IntegrationTests/UITest.cs

```diff
@@ -75,6 +75,9 @@ void HandleApplicationIdle(object sender, EventArgs e)
                 form?.Dispose();
                 Assert.IsEmpty(Application.OpenForms.OfType<T>(), $"{Application.OpenForms.OfType<T>().Count()} open form(s) after test");
             }
+
+            // Await updated async, for instance FileViewer
+            ThreadHelper.JoinPendingOperations();
```

Comment:
This is too late here, should be done right after `await WaitForIdleAsync();` at line 49.
Though I should not have pointed to this function (again): https://github.com/gitextensions/gitextensions/pull/7623/files#r365334811. I was not convinced but voted down.

Reply:
I moved it to line 50 - at least for now.

If the referred discussion is the standard, then all tests with FileViewers need to be changed.
The last failures were all the same test, but previous failures were also in a few other tests.
I rather see stable tests than a winning a few second test time here, but my vote is weak.
@RussKie has the majority vote.

Reply:
The tests fails with this change, moving it to RunFormTest for now (but changing UITest or all forms using FileViewer could still be done in this PR)

Reply:
> I rather see stable tests than a winning a few second test time here, but my vote is weak.
> @RussKie has the majority vote.

You and @mstv have significantly more weight these days :)